models/user.go

```diff
@@ -557,7 +557,9 @@ func (u *User) GetOrgRepositoryIDs() ([]int64, error) {
 	var ids []int64
 	return ids, x.Table("repository").
 		Cols("repository.id").
-		Join("INNER", "team_user", "repository.owner_id = team_user.org_id AND team_user.uid = ?", u.ID).
+		Join("INNER", "team_user", "repository.owner_id = team_user.org_id").
+		Join("INNER", "team_repo", "(team_user.team_id = team_repo.team_id AND repository.id = team_repo.repo_id) OR repository.is_private = 0").
```

Comment:
Check `is_private = 0` first

Reply:
This is not fixed. Please change it to: 
`repository.is_private = 0 OR (team_user.team_id = team_repo.team_id AND repository.id = team_repo.repo_id)`